stale

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+set -Eeuo pipefail
+
+unknown_command() {
+	echo "Unknown command: $*"
+	exit 1
+}
+
+print_help() {
+	echo 'https://github.com/shiftstack/shiftstack-ci'
+	echo
+	echo 'Prints the IDs of the stale resources.'
+	echo
+	echo 'Usage:'
+	echo "$0 <resource_type>"
+	echo
+	echo '"resource_type" can be any openstack resource type.'
+	echo 'Resources are identified as stale if they were last updated more than 5 hours ago.'
+	echo
+	echo 'Examples:'
+	echo "$0 port"
+	echo "$0 'floating ip'"
+}
+
+while getopts h o; do
+	case "$o" in
+		h) print_help; exit     ;;
+		*) unknown_command "$@" ;;
+	esac
+done
+
+declare -r resource_type="$1"
+
+declare \
+	resource_type    \
+	resources        \
+	update_time      \
+	update_timestamp \
+	valid_limit
+
+resources=$(openstack ${resource_type} list -f value -c ID)
+valid_limit="$(date --date='-5 hours' +%s)"
+readonly \
+	resources \
+	valid_limit
+
+for resource_id in $resources; do
+	update_time="$(openstack ${resource_type} show -f value -c updated_at "$resource_id")"
+	update_timestamp=$(date --date="$update_time" +%s)
+	if [[ "$update_timestamp" -lt "$valid_limit" ]]; then
+		echo "${resource_id}"
```

Comment:
Would be nice to also print the name of the resource if it's going to be reviewed by a human.

Reply:
Good point. I’m going to add a flag for that

Reply:
`-n`